tests/test_queries.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+from fsql.api import read_partitioned_table
+from fsql.query import Q_IN
+
+
+def make_test_dfs():
+    df0 = pd.DataFrame(data={"c0": ["part0_c0_r0", "part0_c0_r1"]})
+    df1 = pd.DataFrame(data={"c0": ["part1_c0_r0", "part1_c0_r1"]})
+    df2 = pd.DataFrame(data={"c0": ["part2_c0_r0", "part2_c0_r1"]})
+    return df0, df1, df2
+
+
+def make_path(path: Path):
+    path.mkdir(parents=True)
+    return path
+
+
+def test_q_in(tmp_path):
```

Comment:
Would be useful to extend tests to cover:
- combination of multiple `Q_IN` operators
- simple combination with other operators

Could you add it? It does not have to be complex.

Reply:
good idea, will do